Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+pipeline {
+    agent { label 'go'}
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        OPENSHIFT_API = "https://api.kogito.automation.rhmw.io:6443"
```

Comment:
can we hide these info?
Can we create this envs elsewhere and reuse them like our internal jenkins does?

Reply:
Let me check that!

Reply:
done!

Reply:
please also make sure to clean up your commit so we won't have this history.

Reply:
done!